Auth0/SafariWebAuth.swift

```diff
@@ -196,12 +211,21 @@ class SafariWebAuth: WebAuth {
     }
 
     func handler(_ redirectURL: URL) -> OAuth2Grant {
-        if self.responseType.contains([.code]) {
-            var authentication = Auth0Authentication(clientId: self.clientId, url: self.url, telemetry: self.telemetry)
+        var authentication = Auth0Authentication(clientId: self.clientId, url: self.url, telemetry: self.telemetry)
+        if self.responseType.contains([.code]) { // both Hybrid and Code flow
             authentication.logger = self.logger
-            return PKCE(authentication: authentication, redirectURL: redirectURL, reponseType: self.responseType, nonce: self.nonce)
+            return PKCE(authentication: authentication,
+                        redirectURL: redirectURL,
+                        reponseType: self.responseType,
+                        nonce: self.nonce,
+                        leeway: self.leeway,
+                        maxAge: self.maxAge)
         } else {
-            return ImplicitGrant(responseType: self.responseType, nonce: self.nonce)
+            return ImplicitGrant(authentication: authentication,
```

Comment:
**Question:** Does an implicit grant in this SDK use fragment response type or query?

Reply:
According to the matrix, nope. It doesn't support form_post.

Reply:
https://github.com/auth0/Auth0.swift/blob/04bc1a39875afaa0f724db5bcc502335c6d1896f/Auth0/OAuth2Grant.swift#L72
Looks like it uses fragment response.